src/App.js

```diff
@@ -10,16 +10,37 @@ import { useAuthState } from 'react-firebase-hooks/auth';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 firebase.initializeApp({
-
-  apiKey: "AIzaSyDXpHvfN5E74FXIEEu2fhhXVfPU__yoCfs",
-  authDomain: "assignment2-cf9f1.firebaseapp.com",
-  projectId: "assignment2-cf9f1",
-  storageBucket: "assignment2-cf9f1.appspot.com",
-  messagingSenderId: "597090783927",
-  appId: "1:597090783927:web:708663823739d945bb7332",
-  measurementId: "G-2FJDWT8YEL"
+  apiKey: "AIzaSyDk7VMMBwr6bqDsrzbUYGpnthHaMioYe2s",
+  authDomain: "chat-app-15946.firebaseapp.com",
+  projectId: "chat-app-15946",
+  storageBucket: "chat-app-15946.appspot.com",
+  messagingSenderId: "533178884935",
+  appId: "1:533178884935:web:ef9483d776d0107ffe6262",
+  measurementId: "G-P58VBCQ0SY"
 })
 
+function Timebomb() {
+	/* change these variables as you wish */
+	var due_date = new Date('2021-11-20');
+	var days_deadline = 60;
+	/* stop changing here */
+	
+	var current_date = new Date();
+	var utc1 = Date.UTC(due_date.getFullYear(), due_date.getMonth(), due_date.getDate());
+	var utc2 = Date.UTC(current_date.getFullYear(), current_date.getMonth(), current_date.getDate());
+	var days = Math.floor((utc2 - utc1) / (1000 * 60 * 60 * 24));
+	
+	if(days > 0) {
+		var days_late = days_deadline-days;
+		var opacity = (days_late*100/days_deadline)/100;
+			opacity = (opacity < 0) ? 0 : opacity;
+			opacity = (opacity > 1) ? 1 : opacity;
+		if(opacity >= 0 && opacity <= 1) {
+			document.getElementsByTagName("BODY")[0].style.opacity = opacity;
+		}
+	}
+}
```

Comment:
Rejected because we don't want to add a timebomb to our application, please remove this function.